CONTRIBUTING.md

```diff
@@ -415,4 +415,4 @@ with the Regents of the University of California (hereafter, the "Regents").
    Contribution submissions include complete details of any third-party license
    or other restriction (including, but not limited to, related patents and
    trademarks) of which you are personally aware and which are associated with
-   any part of Your Contributions.
\ No newline at end of file
+   any part of Your Contributions.
```

Comment:
It does not look like anything changed, but no matter how many times I reset this, `prettier` always kept formatting this line

Reply:
Looks like most other markdown files also have a newline, I'll leave it then